### ISSUE TITLE
Discrepancy in FileHandler in OneDrive Shared Library for office extensions

### ISSUE THREAD

**Issue by ivanmaria** [OneDrive/onedrive-api-docs#1339]
#### Category
- [ ] Question
- [ ] Documentation issue
- [x] Bug

#### Expected or Desired Behavior
When a FileHandler is deployed. When we click on the filename in OneDrive Shared Library either all the office extension should open in microsoft application or all the office extension should open via FileHandler (If configured for those extensions) 
#### Observed Behavior
I have created a FileHandler for the file type docx and xlsx. In OneDrive Shared Library when I click on a filename of type .docx, it is opened in Microsoft Word Online. However, when I click on a filename of type .xlsx, it is opened via the FileHandler.

Thank you.

**Comment by sumitmodi84** [comment]
Hi,
Any update on this. I am also facing the same issue.

**Comment by chackman** [comment]
Thanks for your feedback. We would like to clarify whether you have a File Handler registered for .xlsx and .docx?  It sounds like that is the case, but we would like to make sure.

**Comment by sumitmodi84** [comment]
It is registered for both the extensions

**Comment by chackman** [comment]
Thanks, I'll tag for investigating

**Comment by sumitmodi84** [comment]
Also, the issue occurs for shared library only in OneDrive. But for My files, it works as expected.

**Comment by sumitmodi84** [comment]
any update on this....

**Comment by sumitmodi84** [comment]
@chackman I am still facing the issue....and verified the same in mutilple tenants. Were you able to reproduce the same.

**Comment by chackman** [comment]
Sorry, waiting on a partner team to respond, we'll remind them to provide an update.

**Comment by sumitmodi84** [comment]
@chackman has the partner responded?

**Comment by sumitmodi84** [comment]
@chackman waiting for response....still facing the same issue

**Comment by chackman** [comment]
Sorry, still waiting on a partner team to respond, reminding again to provide an update.

**Comment by ivanmaria** [comment]
Hi @chackman, any update?

**Comment by JeremyKelley** [comment]
In the File Handler manifest are you defining actions for the Open action or the Preview action?

**Comment by sumitmodi84** [comment]
@JeremyKelley  It was for Open Action. This issue got fixed in the current product by microsoft.

**Comment by patrick-rodgers** [comment]
As part of a repository clean up effort we are closing older issues. If this issue remains, please: open a new issue, reference this issue, and provide any additional details that may help in resolution. Thank you for your understanding as we work to improve our responsiveness.
